src/PerformanceMonitor/BlockHelper.cs

```diff
@@ -0,0 +1,71 @@
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+
+namespace Neo.Plugins
+{
+    internal static class BlockHelper
+    {
+        /// <summary>
+        /// Returns the time the given block was active
+        /// </summary>
+        /// <param name="block">
+        /// The given block to verify the time
+        /// </param>
+        /// <returns>
+        /// Returns 0 if <paramref name="block"/> is the current block or it is the genesis block or it is null;
+        /// otherwise, returns the time the block was active in milliseconds
+        /// </returns>
+        public static ulong GetTime(this Block block)
```

Comment:
We also need one that return an array of values of times.
We also to have a plugin with that.

@erikzhang removed it in this PR https://github.com/neo-project/neo-modules/commit/68957fb40ff556292318a47e0b06c6c2da728572

The plugin was called `CoreMetrics`.

```cs
        private JObject GetBlocksTime(uint nBlocks, uint lastHeight)
        {
            // It is currently limited to query blocks generated in the last 24hours (86400 seconds)
            uint maxNBlocksPerDay = 86400 / (Blockchain.MillisecondsPerBlock / 1000);
            if (lastHeight != 0)
            {
                if (lastHeight >= Blockchain.Singleton.Height)
                {
                    JObject json = new JObject();
                    return json["error"] = "Requested height to start " + lastHeight + " exceeds " + Blockchain.Singleton.Height;
                }

                if (nBlocks > lastHeight)
                {
                    JObject json = new JObject();
                    return json["error"] = "Requested " + nBlocks + " blocks timestamps is greater than starting at height " + lastHeight;
                }
            }

            if (nBlocks > maxNBlocksPerDay)
            {
                JObject json = new JObject();
                return json["error"] = "Requested number of blocks timestamps exceeds " + maxNBlocksPerDay;
            }

            if (nBlocks >= Blockchain.Singleton.Height)
            {
                JObject json = new JObject();
                return json["error"] = "Requested number of blocks timestamps " + nBlocks + " exceeds quantity of known blocks " + Blockchain.Singleton.Height;
            }

            if (nBlocks == 0)
            {
                JObject json = new JObject();
                return json["error"] = "Requested number of block times can not be = 0";
            }

            JArray array = new JArray();
            uint heightToBegin = lastHeight > 0 ? lastHeight - nBlocks : (Blockchain.Singleton.Height - 1) - nBlocks;
            for (uint i = heightToBegin; i <= heightToBegin + nBlocks; i++)
            {
                Header header = Blockchain.Singleton.GetHeader(i);
                if (header == null) break;

                JObject json = new JObject();
                json["timestamp"] = header.Timestamp;
                json["height"] = i;
                array.Add(json);
            }

            return array;
        }
```